### ISSUE TITLE
latest version of the meteor

### ISSUE THREAD

**Issue by ivansglazunov** [artwells/meteor-accounts-guest#52]
It might be worth to upgrade the package to the latest version of the meteor?

Problems...
```js
Exception in delivering result of invoking 'createUser': TypeError: Cannot read property 'error' of undefined
    at options.userCallback (http://localhost:3000/packages/brettle_accounts-patch-ui.js?7d353a476b21c59b060675fede125d6e13e0ff92:113:14)
    at http://localhost:3000/packages/accounts-base.js?7dabd814506e384c709f8bf707377955f9814129:612:26
    at http://localhost:3000/packages/underscore.js?46eaedbdeb6e71c82af1b16f51c7da4127d6f285:794:19
    at loggedInAndDataReadyCallback (http://localhost:3000/packages/accounts-base.js?7dabd814506e384c709f8bf707377955f9814129:720:5)
    at null._callback (http://localhost:3000/packages/meteor.js?9730f4ff059088b3f7f14c0672d155218a1802d4:999:22)
    at _.extend._maybeInvokeCallback (http://localhost:3000/packages/ddp-client.js?250b63e6c919c5383a0511ee4efbf42bb70a650f:3500:12)
    at _.extend.dataVisible (http://localhost:3000/packages/ddp-client.js?250b63e6c919c5383a0511ee4efbf42bb70a650f:3529:10)
    at http://localhost:3000/packages/ddp-client.js?250b63e6c919c5383a0511ee4efbf42bb70a650f:4365:7
    at Array.forEach (native)
    at Function._.each._.forEach (http://localhost:3000/packages/underscore.js?46eaedbdeb6e71c82af1b16f51c7da4127d6f285:149:11)
```

`random` and `check` problems...

and this...
```
ReferenceError: AccountGuest is not defined
```

**Comment by blackslate** [comment]
Would the fix suggested [here](https://github.com/brettle/meteor-accounts-patch-ui/issues/3) resolve this issue?

**Comment by brettle** [comment]
Yes, that would appear to be the cause. I just published a bug fix release for brettle:accounts-patch-ui which should fix the problem.

**Comment by DrRowland** [comment]
Yes, it's be great if meteor-accounts-guest got an update :)
Brettle, your fix requires --allow-incompatible-update for me...
* kadira:flow-router@=2.7.0 || =2.6.2 || =2.5.0 <- brettle:accounts-patch-ui
0.1.9

**Comment by brettle** [comment]
@DrRowland, what version of `kadira:flow-router` are you using? Can you include the full output you see when attempting to update?

FWIW, the change in the dependencies for `brettle:accounts-patch-ui` occurred in 0.1.9 and was needed to address a [different bug](https://github.com/brettle/meteor-accounts-patch-ui/issues/5). The bug mentioned in the current issue was fixed by 0.1.7, so updating to just `brettle:accounts-patch-ui@0.1.7` (or 0.1.8) should fix it.

**Comment by DuncanRowland** [comment]
cat .meteor/versions | grep kadira:flow-router
kadira:flow-router@2.10.0

Duncans-MBP-2:MyLC Duncan$ meteor add brettle:accounts-patch-ui@0.1.8
 => Errors while adding packages:             
                                              
While selecting package versions:
error: Potentially incompatible change required to top-level dependency: kadira:flow-router 2.7.0, was 2.10.0.
Constraints on package "kadira:flow-router":
* kadira:flow-router@=2.7.0 || =2.6.2 || =2.5.0 <- brettle:accounts-patch-ui 0.1.9

To allow potentially incompatible changes to top-level dependencies, you must pass --allow-incompatible-update on the
command line.

Duncans-MBP-2:MyLC Duncan$ meteor add brettle:accounts-patch-ui@0.1.7
 => Errors while adding packages:             
                                              
While selecting package versions:
error: Potentially incompatible change required to top-level dependency: kadira:flow-router 2.7.0, was 2.10.0.
Constraints on package "kadira:flow-router":
* kadira:flow-router@=2.7.0 || =2.6.2 || =2.5.0 <- brettle:accounts-patch-ui 0.1.9

To allow potentially incompatible changes to top-level dependencies, you must pass --allow-incompatible-update on the
command line.

**Comment by brettle** [comment]
Ah, I see. I'm guessing that your current version of `artwells:accounts-guest` is 0.1.12 or earlier. 0.1.13 added the dependency on `brettle:accounts-patch-ui` and its dependency on particular versions of `kadira:flow-router`. Let me test `brettle:accounts-patch-ui` with `kadira:flow-router@2.10.0`. Assuming it works, I'll release an updated `brettle:accounts-patch-ui` with the dependencies updated.

**Comment by DuncanRowland** [comment]
Thanks, much appreciated :)

**Comment by lpender** [comment]
This should fix: https://github.com/brettle/meteor-accounts-patch-ui/pull/7

**Comment by brettle** [comment]
I just published `brettle:accounts-patch-ui` version 0.1.11 which should fix this.

**Comment by KristerV** [comment]
too long, didn't read, but I don't have any problems in Meteor 1.3 with latest public version.
